src/main/java/ru/hogwarts/school/repositories/StudentRepository.java

```diff
@@ -0,0 +1,16 @@
+package ru.hogwarts.school.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import ru.hogwarts.school.model.Student;
+
+import java.util.Collection;
+import java.util.List;
+
+public interface StudentRepository extends JpaRepository<Student, Long> {
+    static List<Student> findByAgeBetween(Integer min, Integer max) {
+        return StudentRepository.findByAgeBetween(min,max);
+    }
```

Comment:
Этот метод не должен быть статическим. Оставь вот так:
```suggestion
    List<Student> findByAgeBetween(Integer min, Integer max);
```